inspire/base/static/js/dropdown-menu/dropdown-menu.js

```diff
@@ -0,0 +1,150 @@
+ /*
+ * This file is part of INSPIRE.
+ * Copyright (C) 2015 CERN.
+ *
+ * INSPIRE is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * INSPIRE is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * In applying this licence, CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+define([
```

Comment:
This file can be 'beautified' using `jsbeautifier`

Do `$ npm install` in the inspire/ folder then:

```bash
$ grunt jsbeautifier
```

And that will automatically add indentation and nice formatting to all the *.js files. You can discard the changes to the files that you are not interested in.